App.tsx

```diff
@@ -474,12 +476,25 @@ const BaseApp: React.FunctionComponent<{
     );
   }
 
+  const linking = {
```

Comment:
this currently doesnt parse the incoming url into what this expects, will need to add transformation from: 

`https://nwac.us/observations/#/view/observations/48a30020-79a5-4b1f-b895-5fb393a55751`

to

`https://avy/observation/48a30020-79a5-4b1f-b895-5fb393a55751`